pkg/util/volumes/volumes.go

```diff
@@ -1,11 +1,11 @@
-package kubeobjects
```

Comment:
volumes only make sense in context of a pod-spec/template, but `pod` is under `kubeobjects` but `volumes ` is not, why?

Reply:
Thanks for pointing that out, done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/d7ce7d507fe13a29517cf88b4a4f5234f12425e0